sense-android-library/documentation/sensor_profiles.md

```diff
@@ -0,0 +1,472 @@
+  "description": "The proximity to an object in cm",
+  "type": "number",
+}
+~~~
+
+### screen
+*renamed from screen_activity*
+~~~json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "The status of the screen, e.g. on or off",
+  "enum": [ "on", "off"]
+}
+~~~
+*removed inner object with the key-name: screen*
+
+### wifi_scan
+~~~json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "ssid": {
+      "description": "The name of the detected wifi device",
+      "type": "string"
+    },
+    "bssid": {
+      "description": "The mac address of the detected wifi device",
+      "type": "string"
+    },
+    "frequency": {
+      "description": "The signal frequency of the detected wifi device",
+      "type": "integer"
+    },
+    "rssi": {
+      "description": "The signal strength of the detected wifi device",
+      "type": "integer"
+    },
+    "capabilities": {
+      "description": "The capabilities of the detected wifi device, e.g. encryption, WPS",
+      "type": "string"
+    }
+  },
+  "required": [
+    "bssid"  
+  ]
+}
+~~~
+
+
+## Debug sensors
+Debug sensors are used for debuggin the platform libraries.
+
+### app_info
+~~~json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "sense_library_version": {
+      "description": "The version of the sense library",
+      "type": "string"
+    },
+    "app_build": {
+      "description": "Application build (version code, e.g. 1413357349)",
+      "type": "string"
+    },
+    "app_name": {
+      "description": "Application name (e.g. Goalie)",
+      "type": "string"
+    },
+    "app_version": {
+      "description": "Application version (version name, e.g. 3.0.0)",
+      "type": "string"
+    },
+    "locale": {
+      "description": "The device region settings (e.g. en_GB)",
+      "type": "string"
+    },
+    "os": {
+      "description": "OS (e.g Android or iOS)",
+      "type": "string"
+    },
+    "os_version": {
+      "description": "OS Version (e.g. 4.2.2)",
+      "type": "string"
+    },
+    "device_model": {
+      "description": "Device model (e.g. Galaxy Nexus)",
+      "type": "string"
+    },
+  "required": [
+    "sense_library_version",
+    "app_name",
+    "app_build"
+  ]
+}
+~~~
+*The property key sense_platform_version (iOS only) is changed to sense_library_version*
+
+### cortex_log
+~~~json
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "tag": {
+      "description": "Log tag",
+      "type": "string"
+    },
+    "type": {
+      "description": "The type of log information e.g. WARNING",
+      "enum": ["VERBOSE", "DEBUG", "INFO", "WARNING" , "ERROR"]
+    },
+    "text": {
+      "description": "The log message",
+      "type": "string"
+    }
+  },
+  "required": [
+    "tag",
+    "type",
+    "text"
+  ]
+}
+~~~
+
+### time_active
```

Comment:
@tedschmidt why is this as well as sleep and sleep_estimate classified as debug sensors?